src/main/kotlin/org/malachite/estella/commons/EStellaHeaders.kt

```diff
@@ -4,4 +4,5 @@ object EStellaHeaders{
     const val jwtToken = "X-JWT"
     const val refreshToken = "X-Refresh-Token"
     const val authToken = "X-Auth-Token"
+    const val apiKey = "X-API-KEY"
```

Comment:
zmieniłbym nazwę na adminApiKey - na razie nie ma znaczenia, ale trochę lepiej opisuje, co to za key jest. tak samo w nazwie headera na X-Admin-Api-Key